Gruntfile.coffee

```diff
@@ -7,6 +7,9 @@ livereload = require 'tiny-lr'
 path = require 'path'
 fs = require 'fs'
 
+if process.env.LANGUAGE
+  ignoredLanguages = ['./src/languages/!(' + process.env.LANGUAGE + ').coffee']
```

Comment:
It looks like coffeescript supports the following syntax, which is probably a little cleaner.
```
ignoredLanguages = ["./src/languages/!#{process.env.LANGUAGE}.coffee"]
```